### PR TITLE
removed no-var warnings from code

### PR DESCRIPTION
**Pull request by bizob2828**
## Proposed Release Notes
 * Removed warnings from agent for `no-var` eslint rule.

## Details
We still have 65 warnings in tests but wanted to separate them

### CODE REVIEW

File: lib/sampler.js [hunk 1]
```diff
@@ -206,15 +206,15 @@ var sampler = (module.exports = {
       logger.debug('Not adding CPU metric sampler.')
     }
 
-    sampler.state = 'running'
+    this.state = 'running'
```
Comment (michaelgoin) [inline lib/sampler.js:209]: Just calling out that the change to using 'this' directly means a refactor could break the code by changing what 'this' is. Pretty low risk, given this is used only by us and a standard risk in the JavaScript exports we commonly use.

File: lib/util/properties.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ const hasOwnProperty = Object.hasOwnProperty
 
 // The logger needs to be lazy-loaded to get around ordering issues with config.
 let _logger = null
-var getLogger = function makeLogger() {
+let getLogger = function makeLogger() {
   _logger = require('../logger').child({ component: 'util-properties' })
   getLogger = function reallyGetLogger() {
```
Comment (michaelgoin) [inline lib/util/properties.js:14]: 😮

**Review by michaelgoin — APPROVED** [review]
👍

File: lib/sampler.js (hunk 1 above)
Comment (bizob2828) [reply lib/sampler.js:209]: and there are tests for asserting all these cases now so we should be fine. this one was missing so I added a test

File: lib/util/properties.js (hunk 2 above)
Comment (bizob2828) [reply lib/util/properties.js:14]: 🙈

**Pull request merged**